src/reflect/scala/reflect/internal/tpe/FindMembers.scala

```diff
@@ -110,7 +111,7 @@ trait FindMembers {
           //           the component types T1, ..., Tn and the refinement {R }
           //
           //           => private members should be included from T1, ... Tn. (scala/bug#7475)
-          refinementParents :::= currentBaseClass.parentSymbols
+          refinedClasses ::= currentBaseClass
```

Comment:
`walkBaseClasses` is actually called twice from `searchConcreteThenDeferred`, so we'll end up accumulating elements twice in the list of `refinedClasses`. Perhaps it is better to make it a local variable than a field.